ui/app/templates/vault/cluster/access.hbs

```diff
@@ -1,47 +1,59 @@
 <div class="columns">
   {{#menu-sidebar title="Access" class="is-3" data-test-sidebar=true}}
-    <li>
-      {{#link-to "vault.cluster.access.methods" data-test-link=true current-when="vault.cluster.access.methods vault.cluster.access.method"}}
-        Auth Methods
-      {{/link-to}}
-    </li>
-    <li>
-      {{#link-to "vault.cluster.access.identity" "entities" data-test-link=true }}
-        Entities
-      {{/link-to}}
-    </li>
-    <li>
-      {{#link-to "vault.cluster.access.identity" "groups" data-test-link=true }}
-        Groups
-      {{/link-to}}
-    </li>
-    <li>
-      {{#link-to "vault.cluster.access.leases" data-test-link=true}}
-        Leases
-      {{/link-to}}
-    </li>
-    <li>
-      {{#link-to "vault.cluster.access.namespaces" data-test-link=true }}
-        Namespaces
-        {{#unless (has-feature "Namespaces")}}
-          {{#if (is-version "OSS")}}
-            {{edition-badge edition="Enterprise"}}
-          {{/if}}
-        {{/unless}}
-      {{/link-to}}
-    </li>
-    <li>
-      {{#link-to "vault.cluster.access.control-groups" data-test-link=true current-when="vault.cluster.access.control-groups vault.cluster.access.control-group-accessor vault.cluster.access.control-groups-configure"}}
-        Control Groups
-        {{#unless (has-feature "Control Groups")}}
-          {{#if (is-version "OSS")}}
-            {{edition-badge edition="Enterprise"}}
-          {{else}}
-            {{edition-badge edition="Premium"}}
-          {{/if}}
-        {{/unless}}
-      {{/link-to}}
-    </li>
+    {{#if (has-permission "access" "methods")}}
```

Comment:
I initially thought this was checking the multiple permissions. I suppose if I had more context I would recognise that "methods" isn't a type of permission.

I had a look back ^ and saw the second argument is the `routeParams`? A suggestion here would be maybe make a `routeParams=""` attribute/named argument (I think thats the ember term for it) to the helper instead. Not sure I don't have enough context here.

Generally I hit upon the self imposed 'rule' of trying to only ever pass one argument to helpers in Consul UI, anything else needed goes in on an attribute. That keeps it kind of clear what the arguments are.

Here the argument for `has-permission` would be the `permission` and therefore wouldn't need naming. Anything else would be 'named argument'/attribute so a `routeParams="methods"`.

Sorry a bit long winded that, hope you can see what I mean here, ping me if not.